### DISCUSSION TITLE
Support Multi Site

### DISCUSSION THREAD

**Discussion by ivesho** [trendyminds/isolate#37] [Ideas]
Hi there,

I try to run Isolate in multi-site, and restrict users to only specific entries.
My users still can see all entries not specific entries, but when they visit the entry (Whether restrict or not), it will get error message about don't have visit primary site permission.
I think the problem is the setting restricted entries area doesn't have selector to switch sites.

Do you have in mind to support  multi-site?

Thank you.

**Comment by aaronbushnell** [comment]
Hey @ivesho — thanks for the request! Right now there's no plan for multi-site support, unfortunately.

We're always open to new features via pull requests so if you or another individual would like to take a stab at this we certainly don't mind to review it and include it!

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!
